content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -82,22 +81,21 @@ admin      cluster-admins   true
 agent      system:agents    true    
 {{< /highlight >}}
 
-Sensu keeps track of monitored components as entities.
-Let's start by using sensuctl to make sure Sensu hasn't connected to any entities yet:
+Sensu keeps track of monitored components as entities. Start by using sensuctl to make sure Sensu hasn't connected to any entities yet:
 
 {{< highlight shell >}}
 sensuctl entity list
  ID   Class   OS   Subscriptions   Last Seen  
 ──── ─────── ──── ─────────────── ─────────── 
 {{< /highlight >}}
 
-Now we can go ahead and start the Sensu agent to start monitoring the sandbox:
+Now you can start the Sensu agent to start monitoring the sandbox:
```

Comment:
```suggestion
Now you can start the Sensu agent to begin monitoring the sandbox:
```

I have no reason other than start is used twice closely together. Disregarding this suggestion is good as well.